updaterelease.sh

```diff
@@ -358,6 +358,31 @@ update_config_yml() {
   fi
 }
 
+# Generates Javadoc shortlinks for the current snapshot release.
+generate_snapshot_javadoc_shortlinks() {
+  # Creates 2 sub-folders for each non-nested class inside javadocshortcuts.
+  # This ensures that each class documentation is accessible using  https://guava.dev/ClassName
+  # or https://guava.dev/classname.
+  if [[ "$RELEASE" == "snapshot" ]]; then
```

Comment:
Might be easier to read if you invert this.
```suggestion
  if [[ "$RELEASE" != "snapshot" ]]; then
    return
  fi
```